sdk/typescript/src/types/sui-bcs.ts

```diff
@@ -150,10 +76,7 @@ export function isPureArg(arg: any): arg is PureArg {
  * For `Pure` arguments BCS is required. You must encode the values with BCS according
  * to the type required by the called function. Pure accepts only serialized values
  */
-export type CallArg =
```

Comment:
ObjVec is removed in https://github.com/MystenLabs/sui/pull/9043/files#diff-4ebb515e4e2bf9d8ef2f13bc62fc2f729cbb0b3f3bb6c82d2e258c729f3a923f